doc/reference.md

```diff
@@ -275,8 +291,24 @@ constexpr optional<size_t> enum_index() noexcept;
 
   ```cpp
   constexpr auto color_index = magic_enum::enum_index(Color::BLUE);
-  // color_index -> color_index.value() -> 1
-  // color_index -> color_index.has_value() -> true
+  // color_index.value() -> 1
+  // color_index.has_value() -> true
+  ```
+
+```cpp
+template <auto V>
+constexpr size_t enum_index() noexcept;
```

Comment:
documentation bug: [Missing the parameter](https://github.com/Neargye/magic_enum/blob/04a3d32d0fca3d3c79ae4ffde0f002bb4eb38bf8/doc/reference.md?plain=1#L267) from the other variant of `enum_index`